protos/flyteidl/admin/common.proto

```diff
@@ -237,14 +237,24 @@ message Annotations {
     map<string, string> values          = 1;
 }
 
+// Security context fields to define privilege and access control settings
+// In the future a mode (e.g. OVERRIDE, APPEND, etc) can be defined
+// to specify how to merge security context defined at registration and execution time.
+message SecurityContext {
+    // Map of security context values
+    map<string, string> values          = 1;
+}
+
 // Defines permissions associated with executions.
+// Deprecated
```

Comment:
rm?

Reply:
why? This is getting deprecated for securityContext